pandas/tests/test_expressions.py

```diff
@@ -442,3 +441,16 @@ def test_bool_ops_warn_on_arithmetic(self):
                     r = f(df, True)
                     e = fe(df, True)
                     tm.assert_frame_equal(r, e)
+
+    def test_has_bool_ops_column_name_dtype(self, eq, ne):
```

Comment:
Hmm...that's not quite what @jreback was looking for:

https://docs.pytest.org/en/latest/parametrize.html